nengo_gui/static/components/pointer.js

```diff
@@ -178,6 +178,7 @@ Nengo.Pointer.prototype.update = function() {
 
     var items = [];
 
+    /** display the text in proportion to similarity */
```

Comment:
I believe the `/** comment */` style comment is a JavaScript convention similar to docstrings (but I might be wrong). A docstring doesn't apply to a for-loop. So this should be a normal comment (`/* comment */` or `// comment`).